### PR TITLE
add ru and ua translations

### PR DESCRIPTION
**Pull request by BlackYuzia**
I was translated phrase "Players Clips" as "plugin".
I think we don't need repeat again and again a name of plugin, like "this plugin is enabled".

added: ru, ua translations

PS: I hope prefix of messages get unique translation in next versions...

### CODE REVIEW

**Comment by GAMMACASE** [comment]
In that context "Player Clips" stands not for plugin name, but for plugin feature that was enabled, so I don't thing it's correct to name it "plugin".

**Comment by GAMMACASE** [comment]
Also ua translation seems to be in a wrong order.

File: addons/sourcemod/translations/showplayerclips.phrases.txt [hunk 1]
```diff
@@ -1,12 +1,21 @@
 "Phrases"
 {
+	// Комментарий переводчика:
+	// Перевел фразу "Player clips" как "плагин", 
+	// так как не думаю что нет смысла повторяться мол "этот же плагин и включен"
+	// Я думаю ( думать вредно (с) Феникс ) по префиксу (которого нет) и так ясно какой плагин включается
```
Comment (GAMMACASE) [inline addons/sourcemod/translations/showplayerclips.phrases.txt]: And that comment is unnecessary here, as it's main translation file and not a specific for one language.

**Comment by BlackYuzia** [comment]
> Also ua translation seems to be in a wrong order.

ua it's Ukraine, u think UA first order ? ~~УКРАИНА ПОНАД УСЕ~~

> In that context "Player Clips" stands not for plugin name, 

if that, all what I can do, don't remove "Player Clips", I can't translate this words :thinking:

**Comment by GAMMACASE** [comment]
I mean ua translation for playerclips_disabled should be in playerclips_enabled, same for playerclips_enabled translation, it should be in playerclips_disabled.

**Comment by BlackYuzia** [comment]
@GAMMACASE what u mean like a "players clips" function?
The same like a name of plugin, no?

and yep,
I don't know how I can translate "**players clips**" :eyes:

**Comment by GAMMACASE** [comment]
Not sure what to suggest here actually if that is non translatable, can't you leave it as "Player clips"? But "plugin" is wrong here.

File: addons/sourcemod/translations/showplayerclips.phrases.txt [hunk 2]
```diff
@@ -3,10 +3,14 @@
 	"playerclips_disabled"
 	{
 		"en"		"[ShowPlayerClips] Player clips now disabled!"
+		"ru"		"[ShowPlayerClips] Player clips теперь выключен!"
+		"ua"		"[ShowPlayerClips] ПлPlayer clipsагін зараз виключений!"
```
Comment (GAMMACASE) [inline addons/sourcemod/translations/showplayerclips.phrases.txt]: Something went wrong here

**Comment by GAMMACASE** [comment]
Alright, seems good now, thanks

**Pull request merged**